geode-core/src/main/java/org/apache/geode/management/internal/beans/RegionMBeanBridge.java

```diff
@@ -145,6 +145,13 @@ protected RegionMBeanBridge(Region<K, V> region) {
     this.regAttrs = region.getAttributes();
 
     this.isStatisticsEnabled = regAttrs.getStatisticsEnabled();
+    if (isStatisticsEnabled) {
```

Comment:
Actually, disregard my other comment.  Rather than a `try|catch` that is half-expecting an exception to be thrown, why not check the condition that causes the throw directly?

```
    this.isStatisticsEnabled = regAttrs.getStatisticsEnabled();
    // Partitioned regions do not allow for statistics
    if (this.regionAttributesData.getDataPolicy().equals(DataPolicy.PARTITION.toString())
        || this.regionAttributesData.getDataPolicy()
        .equals(DataPolicy.PERSISTENT_PARTITION.toString())){
      this.isStatisticsEnabled = false;
    }
```

Reply:
I don't like this suggestion. I think it's bad practice to dig down into the internals of the object to look for particular behavior - let the object tell you what the behavior is.

I would leave the try/catch as it is.

Reply:
I think the reason for exception thrown is that PartitionedRegion does not implement getStatistics call. This is code that throws that exception:

  @Override
  public CacheStatistics getStatistics() {
    throw new UnsupportedOperationException();
  }